anta/tests/connectivity.py

```diff
@@ -111,49 +113,26 @@ class Input(AntaTest.Input):
         neighbors: list[Neighbor]
         """List of LLDP neighbors."""
 
-        class Neighbor(BaseModel):
```

Comment:
Need to add `ClassVar` for backward compatibility.

Reply:
Added. Thanks!!